src/Raven.Server/Utils/CertificateUtils.cs

```diff
@@ -124,7 +117,7 @@ public void Dispose()
         public static byte[] CreateSelfSignedTestCertificate(string commonNameValue, string issuerName, StringBuilder log = null)
         {
             // Note this is for tests only!
-            CreateCertificateAuthorityCertificate(commonNameValue + " CA", out var ca, out var caSubjectName, log);
+            CreateCertificateAuthorityCertificate(commonNameValue + "-CA", out var ca, out var caSubjectName, log);
```

Comment:
was space problematic?

Reply:
I suspected it may be an issue, yes. I'm not sure, and it is a test only, so would rather not touch